### PR TITLE
Feature/implement friendly match

### PR DESCRIPTION
**Pull request by bryandijkhuizen**
Alles werkt nu wel

### CODE REVIEW

**Review by KevinHerrema — CHANGES_REQUESTED** [review]
it now gets all members for a club when creating a match also the ones not yet approved.

File: Frontend/lib/features/create_match/single_match/player_selector.dart [hunk 1]
```diff
@@ -18,36 +22,63 @@ class _PlayerSelectorState extends State<PlayerSelector> {
...
+        setState(() {
+          if (players.isNotEmpty) {
+            allPlayers = players;
+          } else {
+            allPlayers = [];
+          }
+        });
+      } else {
+        final response = await Supabase.instance.client
+            .rpc('get_club_members', params: {'p_user_id': userId});
```
[the remark this answers is not in the document]
Comment (bryandijkhuizen) [reply Frontend/lib/features/create_match/single_match/player_selector.dart:63]: @wtzb dit is jouw stukje toch?

[the remark this answers is not in the document]
Comment (KevinHerrema) [reply Frontend/lib/features/create_match/single_match/player_selector.dart:63]: al gefixed

**Review by KevinHerrema — APPROVED** [review]
it looks good now and i have tested it on local

**Pull request merged**